sonar-project.properties

```diff
@@ -6,8 +6,7 @@ sonar.tests=src/test/
 sonar.test.exclusions=src/**/*.njk
 sonar.typescript.lcov.reportPaths=coverage/lcov.info
 sonar.javascript.lcov.reportPaths=coverage/lcov.info
-sonar.exclusions=src/**/*.test.ts,**/__mocks__/**,src/**/*content.ts,src/**/components/*.ts,src/**/definition.ts,src/**/summary/util.ts,src/**/summary/util.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/index.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/**/app/controller/PostController.ts,src/**/checkanswers/ConfirmContactDetailsGetController.ts,src/**/respondent/task-list/utils.ts,src/**/respondent/respondentcaseSequence.ts,src/main/steps/common/summary/utils.ts,src/**/applicant/task-list/utils.ts,src/**/components/common/address-history.ts,src/**/ca-da-respondent/task-list/utils.ts,src/main/steps/c100-rebuild/c100sequence.ts,src/main/app/fees/fees-lookup-api.ts,src/main/steps/c100-rebuild/child-details/**/*.ts,src/main/steps/c100-rebuild/other-proceedings/**/*.ts
-sonar.organization=hmcts
```

Comment:
Please add this line back sonar.organization=hmcts as its required for sonar scans